CONTRIBUTING.md

```diff
@@ -27,8 +27,8 @@ This setup is required for testing and deploying this charm. These instructions
 # Import container
 git clone https://github.com/canonical/pgbouncer-container.git
 # Build container locally, since it's not been exported anywhere yet.
-docker build . -t pgbouncer:0.8
-docker save pgbouncer:0.8 -o pgb.tar
+docker build . -t pgbouncer:latest
```

Comment:
Quick question, why latest instead of a specific version.

I can imagine a scenario where a new version is realised and maybe the charm breaks because of the new version. (But also tbh I don't know much about docker and this stuff so this might be an empty concern.)

Reply:
This was just a name picked at random (docker sets its most recent build to `latest` by default), but the tag should ideally the same across local and CI copies since that's what charmcraft expects. I can change this to `pgbouncer:local` or something similar to better communicate this, if that helps?

Reply:
Yes!